src/main.js

```diff
@@ -1,11 +1,11 @@
 import { createApp } from "vue";
 import App from "./App.vue";
-import router from "./router";
-import BaseCard from "./components/ui/BaseCard.vue";
-import BaseButton from "./components/ui/BaseButton.vue";
+
+import { registerPlugins } from "@/plugins";
+import store from "./store/store.js";
+
 const app = createApp(App);
+registerPlugins(app);
 
-app.use(router);
-app.component("base-card", BaseCard);
-app.component("base-button", BaseButton);
+app.use(store);
```

Comment:
should we move this to the registerPlugins(app) method? I see the other freameworks are registered there.

Reply:
fixed